index/server/entrypoint.sh

```diff
@@ -10,7 +11,27 @@ if [ ! -e "$DEVFILE_INDEX" ]; then
     exit 1
 fi
 
+# Start the registry viewer
+npm start &
+
+# Wait for server to start
+until $(curl --output /dev/null --silent --head --fail http://localhost:3000/viewer); do
+    printf '.'
+    sleep 1
+done
+
+# Copy over the registry viewer config
+# cp -rf /viewer-config/devfile-registry-hosts.json /app/config/
+
 # Start the index server
 /registry/index-server
 
-fg %1
+
+# Try to force the registry viewer pages to generate
+# for n in {1..10}; do
+# 	echo "Sleeping $n"
+# 	sleep 1
+# 	curl http://localhost:8080/viewer || true
+# done
```

Comment:
do we need all these uncommented info?

Reply:
D’oh. I knew there was some leftover commented out stuff I needed to remove

Reply:
Fixed.